src/demo/java/org/junitpioneer/jupiter/DefaultLocaleTimezoneExtensionDemo.java

```diff
@@ -43,19 +32,19 @@ void test_with_lanuage() {
 	@Test
 	@DefaultLocale(language = "en")
 	void test_with_lanuage_only() {
-		assertThat(Locale.getDefault()).isEqualTo(createLocale("en"));
+		assertThat(Locale.getDefault()).isEqualTo(PioneerUtils.createLocale("en"));
```

Comment:
I like @Michael1993's idea to move this to `PioneerUtils`, but I'm not sure if it is a good idea to use it in the demos.

Reply:
Demos should use `Locale.Builder`, IMO.